[2024-07-07T11:38:32.963+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T11:38:33.000+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final_exam.create_file_task manual__2024-07-07T11:38:23.357633+00:00 [queued]>
[2024-07-07T11:38:33.016+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final_exam.create_file_task manual__2024-07-07T11:38:23.357633+00:00 [queued]>
[2024-07-07T11:38:33.017+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-07T11:38:33.047+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): create_file_task> on 2024-07-07 11:38:23.357633+00:00
[2024-07-07T11:38:33.056+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=68) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-07T11:38:33.060+0000] {standard_task_runner.py:63} INFO - Started process 72 to run task
[2024-07-07T11:38:33.060+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final_exam', 'create_file_task', 'manual__2024-07-07T11:38:23.357633+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpnrdpd_vo']
[2024-07-07T11:38:33.062+0000] {standard_task_runner.py:91} INFO - Job 80: Subtask create_file_task
[2024-07-07T11:38:33.135+0000] {task_command.py:426} INFO - Running <TaskInstance: Final_exam.create_file_task manual__2024-07-07T11:38:23.357633+00:00 [running]> on host 33ecc1cdac44
[2024-07-07T11:38:33.268+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final_exam' AIRFLOW_CTX_TASK_ID='create_file_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-07T11:38:23.357633+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-07T11:38:23.357633+00:00'
[2024-07-07T11:38:33.270+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-07T11:38:33.271+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-07T11:38:33.272+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'touch /opt/***/dags/database.db']
[2024-07-07T11:38:33.287+0000] {subprocess.py:86} INFO - Output:
[2024-07-07T11:38:33.296+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-07T11:38:33.297+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-07T11:38:33.332+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Final_exam, task_id=create_file_task, run_id=manual__2024-07-07T11:38:23.357633+00:00, execution_date=20240707T113823, start_date=20240707T113833, end_date=20240707T113833
[2024-07-07T11:38:33.396+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-07T11:38:33.426+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-07T11:38:33.431+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
